### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by barborapilna**
Hello, I am sending you my version of refactoring to check... Thank you.

### CODE REVIEW

File: INPTPZ1/INPTPZ1.csproj.user [hunk 1]
```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (rodi0878) [inline INPTPZ1/INPTPZ1.csproj.user]: .user files contains user configuration and thus shoudnt be added to repo

File: INPTPZ1/NewtonsFractalSolver.cs [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+        private double yStep;
+
+        private int imageWidth;
+        private int imageHeight;
+
+        private Polynomial polynomial;
+        private Polynomial derivation;
+        private List<ComplexNumber> roots;
+
+        public void ParseArguments(string[] args)
```
Comment (rodi0878) [inline INPTPZ1/NewtonsFractalSolver.cs:37]: too long method

File: INPTPZ1/NewtonsFractalSolver.cs [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+            if (!known)
+            {
+                roots.Add(coordinates);
+                numberOfRoots = roots.Count;
+            }
+
+            return numberOfRoots;
+        }
+
+        private readonly Color[] colors = new Color[]
```
Comment (rodi0878) [inline INPTPZ1/NewtonsFractalSolver.cs]: mixing methods and attributes is poor organization of code, move attributes to the top of class

**Review by rodi0878 — CHANGES_REQUESTED** [review]
Please fix metioned errors before merging of PR

File: INPTPZ1/NewtonsFractalSolver.cs (hunk 2 above)
Comment (barborapilna) [reply INPTPZ1/NewtonsFractalSolver.cs:37]: method divided

File: INPTPZ1/INPTPZ1.csproj.user (hunk 1 above)
Comment (barborapilna) [reply INPTPZ1/INPTPZ1.csproj.user]: file deleted

File: INPTPZ1/NewtonsFractalSolver.cs (hunk 3 above)
Comment (barborapilna) [reply INPTPZ1/NewtonsFractalSolver.cs]: attribute moved to another attributes

**Comment by rodi0878** [comment]
Accepted

**Pull request closed without merging**